JavaScript/Ex-1.js

```diff
@@ -0,0 +1,7 @@
+//малюємо стільки котів, скільки захочемо!
```

Comment:
this code relates just for the first part of the book. But where are the other samples?